tensorflow/contrib/rnn/BUILD

```diff
@@ -346,6 +348,8 @@ tf_kernel_library(
     srcs = [
         "kernels/blas_gemm.cc",
         "kernels/blas_gemm.h",
+        "kernels/lstm_ops.h",
```

Comment:
You shouldn't add these here. Instead add the dependencies
`"gru_ops_op_lib"` and `"lstm_ops_op_lib"` to the target